added a method to add text to the top of the code

i164j9 opened the pull request:
added a font file from: https://fonts.google.com/specimen/Roboto+Flex?query=Roboto
changed main() to give the option to have text or not by changing commented lines
text is only added when there is a font file specified
renamed sample Aztec code and added a second sample

i164j9 committed:
added a method to add text to the top of the code
added a font file from: https://fonts.google.com/specimen/Roboto+Flex?query=Roboto
changed main() to give the option to have text or not
by changing commented lines
text is only added when there is a font file specified
renamed sample aztec code and added a second sample

i164j9 committed:
fixed comments

delimitry commented on aztec_code_generator.py:
```diff
@@ -544,7 +544,7 @@ class AztecCode(object):
     """
 
-    def __init__(self, data, size=None, compact=None):
+    def __init__(self, data, fontfile=None, size=None, compact=None):
```
Please add fontfile to docstring.

delimitry commented on aztec_code_generator.py:
```diff
@@ -594,6 +595,13 @@ def save(self, filename, module_size=1):
                     (x * module_size, y * module_size,
                      x * module_size + module_size, y * module_size + module_size),
                     fill=(0, 0, 0) if self.matrix[y][x] == '#' else (255, 255, 255))
+        if self.my_font != None:
```
Better to use `is not None:`

delimitry commented on aztec_code_generator.py:
```diff
@@ -840,16 +848,72 @@ def __encode_data(self):
         data_cw_count = self.__add_data(self.data)
         self.__add_mode_info(data_cw_count)
 
+    def add_text(self, img_obj, text, module_size = 4):
```
Please format the code and add arguments to docstring.

delimitry commented:
.

delimitry commented on README.md:
````diff
@@ -10,6 +10,8 @@ PIL - Python Imaging Library (or Pillow)
 ```python
 data = 'Aztec Code 2D :)'
 aztec_code = AztecCode(data)
+#to add text:
+aztec_code = AztecCode(data,fontfile='path/to/your/font.ttf')
````
Please format this lines according to pep8.

i164j9 committed:
renamed font file: font/RobotoFlex-VariableFont.ttf
fixed docstring
fixed line 600 - if self.my_font is not None:
fixed formatting in README.md

i164j9 committed:
fixed formatting and added parameters to docstring
